src/conversations/Conversation.ts

```diff
@@ -41,6 +41,10 @@ export class ConversationV1 {
     return this.client.listConversationMessages(this.peerAddress, opts)
   }
 
+  get topic(): string {
+    return buildDirectMessageTopic(this.peerAddress, this.client.address)
```

Comment:
Exposing the `topic` field on the V1 conversation, mostly for consistency's sake.